code-challenges.test.js

```diff
@@ -48,5 +135,46 @@ const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
+it("takes in an array of 5 numbers and determines whether or not the array is a Full House", () => {
+  expect(isFullHouse(hand1)).toEqual(true)
+  expect(isFullHouse(hand2)).toEqual(false)
+  expect(isFullHouse(hand3)).toEqual(false)
+  expect(isFullHouse(hand4)).toEqual(true)
+})
+})
+
+// ReferenceError: fullHouse is not defined
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 2 passed, 3 total
+
+// This is a good fail. Now we'll define the method.
+
+// b) Create the function that makes the test pass.
 
-// b) Create the function that makes the test pass.
\ No newline at end of file
+// INPUT: array of 5 numbers [5, 5, 5, 3, 3]
+// OUTPUT: fullHouse = true 'Your hand is a Full House'
+//         fullHouse = false 'Your hand is not a Full House'
+
+// We have to declare the function and name it 'isFullHouse' that takes in an array of cards.
```

Comment:
Excellent work in this entire section!  I really appreciate the extremely thorough pseudocode you provide in all of your work.  👏